sites/cheerpx/src/content/docs/13-tutorials/simple-executable.mdx

```diff
@@ -50,12 +50,13 @@ gcc -static -m32 -o hello_cheerpx hello_cheerpx.c
 If you cannot compile directly on your system, use Docker to create a 32-bit statically linked executable. Open your terminal, navigate to your project directory, and run the following command:
 
 ```sh
-docker run --rm -v "$(pwd)":/src -w /src i386/debian bash -c "apt-get update && apt-get install -y gcc && gcc -static -m32 -o hello_cheerpx hello_cheerpx.c"
+docker run --rm --platform=linux/amd64 -v "$(pwd)":/src -w /src i386/debian bash -c "apt-get update && apt-get install -y gcc && gcc -static -m32 -o hello_cheerpx hello_cheerpx.c"
 ```
 
 Explanation:
 
 - `docker run --rm`: Runs a Docker container and removes it after execution.
+- `--platform=linux/amd64`: Defines the platform to use inside the container.
```

Comment:
This does not seem to be accurate. Inside the container the platform must be 32-bit x86, not 64-bit x86. The purpose of the option is to enable execution of x86 containers on ARM M-class processors.

@AtibQur Please restructure this suggestion to clarify that only MacOS users should add this option. It does not seem advisable to enable this across the board.